client/src/components/tests/Hover.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Enzyme, { shallow, mount } from 'enzyme';
+
+import Hover from '../Hover.jsx';
+
+
+describe('Hover', () => {
+
+  const props = {
+    tags: [
+      {
+        symbols: ['', '', '', '', ''],
+        price: ['', '', '', '', ''],
+      },
```

Comment:
Good idea to leave some defaults in case nothing is fetched from the database, the client component will still load. Does anything strange happen with your typography when you use empty strings? Maybe consider using some mock prices that resemble your actual data.